living_documentation_regime/model/consolidated_issue.py

```diff
@@ -200,18 +195,13 @@ def generate_directory_path(self, issue_table: str) -> list[str]:
             # If no label ends with "Topic", create a "NoTopic" issue directory path
             if not topic_labels:
                 self.__topics = ["NoTopic"]
-                logger.error("No Topic label found for Issue #%i: %s (%s)", self.number, self.title, self.repository_id)
+                self.__errors.update({"TopicError": "No Topic label found."})
```

Comment:
Missing context where it was not found.

Reply:
Please see comment: https://github.com/AbsaOSS/living-doc-generator/pull/79#discussion_r1949080083